fluent-gecko/xpcom_config.js

```diff
@@ -17,14 +21,6 @@ export default {
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */\n\n`,
-
-  intro: `/* fluent@${version} */`,
-  preferConst: true,
-  context: 'this',
-  output: {
-    format: 'es',
-  },
-  acron: {
```

Comment:
This was misspelled; it should read `acorn`. I guess it wasn't needed after all :)